fixtures/gravity/index.js

```diff
@@ -0,0 +1,106 @@
+// @flow
+import "babel-polyfill"
+
+import { spawnSync } from "child_process"
+import { basename, dirname, extname, join } from "path"
+import { writeFile } from "fs"
+
+import request from "request-promise"
```

Comment:
Ah cool, I’ll take a look at that when continuing work on the script, like e.g. record positron responses.